state_solver/suspension.py

```diff
@@ -87,31 +97,34 @@ def __get_inclination_angle(self, tire_name:str, tire:engine.Tire, steered_angle
         
         return tire.static_camber + steering_induced + roll_induced + heave_induced + pitch_induced
           
-    def __get_tire_output(self, tire_name:str, tire:engine.Tire, normal_force:float, slip_angle:float,
-                        inclination_angle:float, steering_slip:float):
+    def __get_tire_output(self, tire_name:str, tire:Tire, normal_force:float, slip_angle:float,
+                        inclination_angle:float, steering_slip:float, slip_ratio:list):
 
-        lateral_force = tire.lateral_pacejka(inclination_angle, normal_force, slip_angle)
-        tire_centric_forces = np.array([0, lateral_force, normal_force])
+        tire_centric_forces = tire.comstock(slip_ratio, slip_angle, normal_force, inclination_angle)
```

Comment:
Realize it's somewhat implied, but can we name it something like get_comstock_forces just to be explicit this isn't the parameter set for the tire?

Reply:
@rhorvath02